src/assets/css/main.css

```diff
@@ -11,8 +11,8 @@ html {
   --light-grey: #999999;
 
   /* Sizes */
-  --footer-height: 7em;
-  --navbar-height: 4.5em;
+  --footer-height: 5.5em;
+  --navbar-height: 3em;
```

Comment:
Personally not a fan of the smaller navbar size - it makes the profile picture look out of place, and shrinking the picture would make it a bit too small imo
![image](https://user-images.githubusercontent.com/39015447/86123936-51ed4e00-baa8-11ea-90b6-505a64b4dd6e.png)

Reply:
I know that Arvind wanted something which emulated a thinner navbar like Facebook's. I think **3.5em** for the navbar does that. The footer is fine, I think.